security/infisical/Portfile

```
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/infisical/infisical 0.28.4 infisical-cli/v
+github.tarball_from archive
+revision            0
+
+categories          security devel
+maintainers         openmaintainer {esafak @esafak}
+license             MIT
+
+description         Official CLI for Infisical, an open source secrets platform
+long_description    ${description}
+homepage            https://infisical.com/
+
+# Allow Go to fetch deps at build time
+go.offline_build    no
+use_configure       no
+
+build.dir           ${worksrcpath}/cli
+build.cmd           ${go.bin} build
+build.args          -o ${name} -ldflags=\"-X 'github.com/Infisical/infisical-merge/packages/util.CLI_VERSION=${go.version}'\"
```

What you have right now works, but you can probably drop `build.dir` and tell go to build that dir with the `build.args`:

```suggestion
build.args          -o ${name} -ldflags=\"-X 'github.com/Infisical/infisical-merge/packages/util.CLI_VERSION=${go.version}' ./cli\"
```

---

If you do drop `build.dir` and go this path, you'll need to adjust your destroot phase to install the binary from `${worksrcdir}` instead of `${worksrcdir}/cli`.